var/spack/repos/builtin/packages/typst/package.py

```diff
@@ -19,6 +19,7 @@ class Typst(CargoPackage):
     license("Apache-2.0", checked_by="upsj")
 
     version("0.12.0", commit="737895d769188f6fc154523e67a9102bc24c872e", tag="v0.12.0")
+    version("0.13.0", commit="8dce676dcd691f75696719e0480cd619829846a9", tag="v0.13.0")
```

Comment:
```suggestion
    version("0.13.0", commit="8dce676dcd691f75696719e0480cd619829846a9", tag="v0.13.0")
    version("0.12.0", commit="737895d769188f6fc154523e67a9102bc24c872e", tag="v0.12.0")
```

@upsj one quick nitpick (typically newer versions are placed above older versions) other than that this looks good.